### DISCUSSION TITLE
AttributeError: 'HeteroData' has no attribute 'edge_index' AddLaplacianEigenvectorPE

### DISCUSSION THREAD

**Discussion by 1AngelUS** [pyg-team/pytorch_geometric#9382] [General]
Hi, 

I am generating the positional encodings for the heterogeneous graph. 

I am getting this error: AttributeError: 'HeteroData' has no attribute 'edge_index' AddLaplacianEigenvectorPE 

These encodings do not have support for hetero data?

Thank you!

**Comment by rusty1s** [comment]
Currently, `AddLaplacianEigenvectorPE` does not support `HeteroData`, sorry :(
